src/openrpc/advertising.json

```diff
@@ -322,6 +375,35 @@
             "description": "The authentication provider, when it is separate entity than the app provider, e.g. an MVPD."
           }
         }
+      },
+      "AdvertisingIdOptions": {
+        "title": "AdvertisingIdOptions",
+        "type": "object",
+        "properties": {
+          "scope": {
+            "type": "object",
+            "description": "Provides the options to send scope type and id to select desired advertising id",
+            "required": [
+              "type",
+              "id"
+            ],
+            "properties": {
+              "type": {
+                "type": "string",
+                "enum": [
+                  "browse",
+                  "content"
+                ],
+                "default": "browse",
+                "description": "The scope type, which will determine where to show advertise"
+              },
+              "id": {
+                "type": "string",
+                "description": "The scope id to determine if the advertising is app based"
```

Comment:
Think this description should be like:
"A value that identifies a specific scope within the scope type"